### PR TITLE
Move Settings to the sidebar

### PR DESCRIPTION
**Pull request by j-f1**
Part of my next batch of sidebar tweaks! This makes the settings UI more visible, and coalescing the “More” section of the sidebar as where you go to configure your budget.

### CODE REVIEW

File: packages/loot-design/src/components/sidebar.js [hunk 1]
```diff
@@ -541,6 +537,12 @@ function Tools() {
             to="/rules"
             indent={15}
           />
+          <SecondaryItem
```
Comment (MatissJanis) [inline packages/loot-design/src/components/sidebar.js:540]: Could we have it below "repair split transactions"? IMO it should be last item in this menu

Comment (j-f1) [reply packages/loot-design/src/components/sidebar.js:540]: It will be after #456 :)

**Comment by rich-howell** [comment]
Does this get superceeded by this https://github.com/actualbudget/actual/pull/459 or are we still planning to move Settings from that context menu?

**Comment by j-f1** [comment]
I would still like to move the Settings button to the sidebar, I just temporarily moved it in that PR because there was nowhere else for it to go.

**Comment by trevdor** [comment]
🤔 Just thinking out loud...
That kebab menu is getting kinda sparse. 

What if we yank Help out of it, too, and make it very discoverable down, say, in like a fixed footer (long accounts lists could scroll behind it)? 
Screenshot 2023-01-16 at 12 38 41 PM

Then `Close Budget` could be done with a simple X icon, or some other way.

**Comment by MatissJanis** [comment]
I'm happy to approve this, but it has some conflicts. Would you mind doing a second pass?

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**